cilium-cli/cli/connectivity.go

```diff
@@ -75,6 +75,20 @@ func RunE(hooks api.Hooks) func(cmd *cobra.Command, args []string) error {
 			}
 		}
 
+		if params.PrintImageArtifacts {
+			if cmd.Use == "test" {
+				fmt.Fprintln(params.Writer, params.CurlImage)
```

Comment:
Let me know if I should move the images to a separate struct and then use this one within the `Parameters` struct to use a loop here instead of multiple manual `fmt.Fprintln` statements.

Reply:
If we imagine that `&params.Images` is a `*map[string]string]`, then we could iterate like this:

```
for component, imagePath := &params.Images {
    fmt.Fprintln(params.Writer, fmt.Sprintf("%s: %s", component, imagePath)
}
```

However, in order for that to function, the parameter parsing would need to be dynamic, maybe based on a manifest file. For the purposes of this imagePath printing functionality, keeping the existing explicit parameters for the image paths, and explicitly printing them here is fine, and limits the scope of any potential side effects.